src/frontend/Komponenter/Migrering/MigrerFagsak.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo, useState } from 'react';
+import { useApp } from '../../App/context/AppContext';
+import {
+    byggSuksessRessurs,
+    byggTomRessurs,
+    Ressurs,
+    RessursStatus,
+} from '../../App/typer/ressurs';
+import { MigreringInfoResponse } from '../../App/typer/migrering';
+import { AxiosRequestConfig } from 'axios';
+import { Knapp } from 'nav-frontend-knapper';
+import DataViewer from '../../Felles/DataViewer/DataViewer';
+import {
+    formaterNullableMånedÅr,
+    formaterTallMedTusenSkille,
+    nullableBooleanTilTekst,
+} from '../../App/utils/formatter';
+import Utregningstabell from '../Behandling/VedtakOgBeregning/InnvilgeVedtak/Utregningstabell';
+import { useToggles } from '../../App/context/TogglesContext';
+import { ToggleName } from '../../App/context/toggles';
+
+const MigrerFagsak: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
+    const { axiosRequest } = useApp();
+    const { toggles } = useToggles();
+    const [migreringInfo, settMigreringInfo] = useState<Ressurs<MigreringInfoResponse>>(
+        byggTomRessurs()
+    );
+    const [migrertStatus, settMigrertStatus] = useState<Ressurs<string>>(byggTomRessurs());
+
+    const hentMigreringConfig: AxiosRequestConfig = useMemo(
```

Comment:
Stopper toggle disse? Gjør kanskje ikke noe om vi gjør kall?

Reply:
Trooor det, men ville testet ut i preprod for å være sikker :) Det gjør vel ikke noe om disse kallene gjøres uansett nei, det eneste som har noe å si er vel om man klikker på "Migrer fagsak" knappen

Reply:
Sånn sett hadde det kanskje holdt å kun migrere fagsakknappen? Eller..

Reply:
Dette er kun config, selve hentingen utføres lengre ned (via klikk på knapp)